### ISSUE TITLE
Highlight HTML inside JavaScript strings

### ISSUE THREAD

**Issue by saurabhdaware** [abelljs/vscode-abell-language-features#3]
```html
    <span class="articles-container">
    {{
      articles.map(article => ( /* html */
        `<a href="${article.url}" class="article shadow">${article.name}</a>`
        )).join('<br/>');
    }}
    </span>
```

In the snippet above, we should be able to highlight anchor tag HTML inside map function

**Comment by anuraghazra** [comment]
Take a look at this ->

https://marketplace.visualstudio.com/items?itemName=Tobermory.es6-string-html

**Comment by saurabhdaware** [comment]
Yes I was referring to that but I am an absolute noob at this and I didn't understand a single thing so I'll just hope someone else contributes to this or maybe I'll explore after getting some time from abell :c

**Comment by saurabhdaware** [comment]
Resolved in afe5c0ffb977682568fe8debaf92c72fa4ddc3bd
